WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Split Shipments/WooShippingSplitShipmentsViewModel.swift

```diff
@@ -78,8 +78,10 @@ final class WooShippingSplitShipmentsViewModel: ObservableObject {
 
     var topTabItems: [TopTabItem<EmptyView>] {
         shipments.enumerated().map { (index, item) in
-            TopTabItem(name: String.localizedStringWithFormat(Localization.shipmentFormat, index + 1),
-                       content: { EmptyView() })
+            let purchasedIcon = UIImage(systemName: "checkmark.circle.fill")?.withRenderingMode(.alwaysTemplate)
+            return TopTabItem(name: String.localizedStringWithFormat(Localization.shipmentFormat, index + 1),
+                              icon: item.isPurchased ? purchasedIcon : nil,
+                              content: { EmptyView() })
```

Comment:
nit: We could initialise the icon outside of the `map` to avoid creating the image for every shipment. 

```suggestion
        let purchasedIcon = UIImage(systemName: "checkmark.circle.fill")?.withRenderingMode(.alwaysTemplate)
        return shipments.enumerated().map { (index, item) in
            TopTabItem(name: String.localizedStringWithFormat(Localization.shipmentFormat, index + 1),
                       icon: item.isPurchased ? purchasedIcon : nil,
                       content: { EmptyView() })
        }
```

Reply:
Done in 8a33bbba27d25ec3252855e802632b998b1b004e.